workers/data_refinery_workers/processors/salmon.py

```diff
@@ -197,7 +197,7 @@ def _find_or_download_index(job_context: Dict) -> Dict:
     except FileExistsError:
         # Someone already installed the index or is doing so now.
         pass
-    except e:
+    except Exception as e:
```

Comment:
This issue also crops up in utils.py. If you get that one as well, I think this should close #569.